test/test.js

```diff
@@ -1545,6 +1545,22 @@ exports['wrap EventEmitter (or jQuery) on handler with args wrapping by array']
     });
 };
 
+exports['takeUntil'] = function (test) {
+    a = _();
+    b = _();
+    a.resume();
```

Comment:
then to array should call twice right? the first time it will be be `[1]` and the second will be `[1,2]` right?